build/python/build/project.py

```diff
@@ -52,6 +52,10 @@ def unpack(self, toolchain, out_of_tree=True):
         path = untar(self.download(toolchain), parent_path, self.base)
         if self.patches is not None:
             push_all(toolchain, path, self.patches)
+        if os.path.isdir(path) is not True:
+            # rename wolfssl-4.0.0-stable to wolfssl-4.0.0
```

Comment:
Bad and fragile kludge for exactly one special cae.

Reply:
I agree. wolfssl source code is gotten from 'https://fossies.org/linux/misc/wolfssl-4.0.0.tar.gz' as a primary source. It no longer keeps the corresponding version. The second choice, https://github.com/wolfSSL/wolfssl/archive/v4.0.0-stable.tar.gz' is expanded as wolfssl-4.0.0-stable which does not match following processing. Do you have better idea?